pumble-sdk/src/api/ApiClient.ts

```diff
@@ -16,6 +17,7 @@ export class ApiClient {
         users: UsersApiClientV1;
         workspace: WorkspaceApiClientV1;
         calls: CallsApiClientV1;
+        app: AppClientV1;
```

Comment:
should we name it "AppApiClientV1" since all others end with "ApiClientV1"?

Reply:
AppApi... not quite sure about this name really, it's kinda mouthful